packages/core/utils/src/policy.ts

```diff
@@ -152,10 +38,4 @@ const createPolicyContext = (type: string, ctx: object) => {
   );
 };
 
-export {
```

Comment:
Why do we keep some policy-related utils in there?

Reply:
eventually `import policyDomain from '@strapi/core/policy'` but for now I don't want to put everything in the core. not the goal of the PR